core/src/processing/opengl/PGL.java

```diff
@@ -1883,23 +1885,20 @@ protected static int qualityToSamples(int quality) {
       // We need to replace 'texture' uniform by 'texMap' uniform and
       // 'textureXXX()' functions by 'texture()' functions. Order of these
       // replacements is important to prevent collisions between these two.
-      String[] search = new String[] {
-          "varying", "attribute",
-          "texture",
-          "texMap2DRect", "texMap2D", "texMap3D", "texMapCube",
-          "gl_FragColor"
+      Pattern[] search = new Pattern[] {
+          Pattern.compile(String.format(GLSL_ID_REGEX, "varying|attribute")),
+          Pattern.compile(String.format(GLSL_ID_REGEX, "texture")),
+          Pattern.compile(String.format(GLSL_FN_REGEX, "textureRect|texture2D|texture3D|textureCube")),
+          Pattern.compile(String.format(GLSL_ID_REGEX, "gl_FragColor"))
       };
       String[] replace = new String[] {
-          "in", "in",
-          "texMap",
-          "texture", "texture", "texture", "texture",
-          "fragColor"
+          "in", "texMap", "texture", "_fragColor"
```

Comment:
To make it more unique. Somebody could already have fragColor variable in their code and we don't have collision detection yet.